test/cmdlineTests/name_simplifier/output

```diff
@@ -83,7 +83,7 @@ object "C_59" {
                 if or(gt(newFreePtr, 0xffffffffffffffff), lt(newFreePtr, memPtr)) { panic_error_0x41() }
                 mstore(64, newFreePtr)
             }
-            function copy_literal_to_memory_64902fd228f7ef267f3b474dd6ef84bae434cf5546eee948e7ca26df3eda1927() -> memPtr
+            function copy_literal_to_memory2fd228f7ef267f3b474dd6ef84bae434cf5546eee948e7ca26df3eda1927() -> memPtr
```

Comment:
Perhaps retain the underscore after memory and before the literal?

Reply:
Ah, no, my regex is wrong. It removes decimal characters that are followed by a non-hex character, so for example another decimal character.